src/components/dashboard/ModularDashboard.tsx

```
@@ -16,9 +16,10 @@ import {
   AlertTriangle, 
   Bot, 
   TrendingUp, 
-  Bell, 
+  Bell,
   Settings,
   MapPin,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)